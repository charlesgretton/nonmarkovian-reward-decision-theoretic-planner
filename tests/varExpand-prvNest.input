loadWorld("prvNest.pltl")
PLTLvarExpand
actionToDot("a", "(prv^2 a)")
quit
