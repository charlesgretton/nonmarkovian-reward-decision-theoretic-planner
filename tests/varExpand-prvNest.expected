

digraph "Effect of action `a' on proposition `(prv^2 a)'" {
center = true;
edge [dir = none];
n0 [label="(prv a)"];
n0 -> n1;
n0 -> n2 [style=dashed]
n1 [label="1.00" shape=box];
n2 [label="0.00" shape=box];
}


